packages/redis/redis.conf

```diff
@@ -0,0 +1,5 @@
+# Directory to store dump.rdb
+dir /app/redis
```

Comment:
Shouldn't we put this in ``/data/redis`` ?
IMO Redis should be persisted

Reply:
![image](https://github.com/homarr-labs/homarr/assets/63781622/e91b518e-62d1-4ecc-8f21-d40eccc00115)
Moved it to `/appdata/redis`